AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -731,7 +721,6 @@ - (void)testDontShowConfirmationAlertIfNoMandatoryReleaseWhileNoNetwork {
   // Then
   [self waitForExpectationsWithTimeout:1
                                handler:^(NSError *error) {
-
```

Comment:
Empty lines before comments shouldn't be removed

Reply:
All empty lines are where they need to be, no idea why it shows up like this on here. 🤷‍♂️